script/mail_new_activity.rb

```diff
@@ -142,8 +142,7 @@ def story_subject(story, prefix = "")
 
   # repeat for comments
 
-  last_comment_id = (Keystore.value_for(LAST_COMMENT_KEY) ||
-    Comment.last.id).to_i
+  last_comment_id = (Keystore.value_for(LAST_COMMENT_KEY) || Comment.last && Comment.last.id).to_i
```

Comment:
Same comment here with the safe navigation operator.

```suggestion
  last_comment_id = (Keystore.value_for(LAST_COMMENT_KEY) || Comment.last&.id).to_i
```